wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceRuntimes.java

```diff
@@ -129,8 +133,7 @@ public void validate(Environment environment)
     if (!infraByRecipe.containsKey(type)) {
       throw new NotFoundException("Infrastructure not found for type: " + type);
     }
-
-    infraByRecipe.get(type).estimate(environment);
+    createInternalEnvironment(environment);
```

Comment:
Can you add comment that explains that purpose of this call is validation which is performed in it

Reply:
Added comment